nservicebus/messaging/delayed-delivery.md

```diff
@@ -5,10 +5,21 @@ tags:
 - Defer
 ---
 
-When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this was achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
+When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this could be achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
```

Comment:
Version X should be capitalized.

Rather than "Before Version 6" are we better off saying "In Version 5 and below"? This gets rid of the past tense "was/could be" as well. It's still how Version 5 and below works.